app/src/main/java/com/example/todokotlinandroid/CategoriesEmptyScreen.kt

```diff
@@ -0,0 +1,76 @@
+package com.example.todokotlinandroid
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.TextUnitType
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Preview
+@Composable
+
+fun CategoriesEmptyScreen() {
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+    ) {
+        Row(
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(
+                    start = dimensionResource(id = R.dimen.default_margin),
+                    top = dimensionResource(id = R.dimen.default_margin)
+                )
+        ) {
+            Text(
+                text = "Categories",
+                fontWeight = FontWeight.Bold
+            )
+
+            Text(
+                text = "+",
+                fontWeight = FontWeight.Bold
+            )
+        }
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+        ) {
+            Text(
+                text = "Nothing here",
+                color = Color.LightGray,
+                modifier = Modifier
+                    .align(Alignment.Center)
+            )
+            Button(
```

Comment:
Button is missing content attribute - this is why you can't compile and run project

Reply:
You can do this by adding `content = {}` argument or just {} after last parenthesis on Button function call
e.g. 
```
Button(...) {
}
```